CONTRIBUTING.md

```diff
@@ -1,59 +1,56 @@
-# Contributing
+# Contribuindo
 
-*Please note that this project is released with a [Contributor Code of Conduct](CODE_OF_CONDUCT.md). By participating in this project, you agree to abide by its terms.*
+______________________________________
 
-Since Diário Oficial seeks to become a trustful source of official governmental announcements, every data source must also be trustful. A private company publishing government gazettes without official permission, for instance, is not a trustful source.
+_[Click here](languages/en-US/CONTRIBUTING.md) to read this article in english._
```

Comment:
As I comment in another PR. I would like to move the `language` directory into a `docs` directory. Just to keep all the docs in an organized directory structure.